.github/workflows/rspec.yml

```diff
@@ -91,3 +92,12 @@ jobs:
         with:
           name: failed-browser-tests
           path: tmp/screenshots
+  deploy:
+    name: Deploy Staging To fly.io
+    runs-on: ubuntu-latest
+    needs: rspec
+    if: github.ref == 'refs/heads/main'
+    steps:
+      - uses: actions/checkout@v2
+      - uses: superfly/flyctl-actions/setup-flyctl@master
+      - run: flyctl deploy --remote-only --build-arg RAILS_ENV=staging -a human-essentials-staging
```

Comment:
Uh this shouldn't be part of the rspec workflow, no? Or at least we should rename it?